packages/office-ui-fabric-react/src/components/Announced/examples/Announced.BulkOperations.Example.tsx

```diff
@@ -131,12 +97,12 @@ export class AnnouncedBulkOperationsExample extends React.Component<
           Note: This example is to showcase the concept of copying, uploading, or moving many items and not fully illustrative of the real
           world scenario.
         </Text>
-        {this._renderAnnounced()}
+        {numberOfItems > 0 && <Announced message={`${numberOfItems} item${numberOfItems === 1 ? '' : 's'} moved`} />}
```

Comment:
Just wondering why you took this logic out of the `_renderAnnounced` method, as it looks like you kept it in two other examples. I originally included the method to create some consistency across all the examples. Did you find it to be unnecessary?

It could be better this way, but I was curious to hear your thoughts!